build.sbt

```diff
@@ -3,6 +3,7 @@ import akka.projections.Dependencies
 lazy val core =
   Project(id = "akka-projection-core", base = file("akka-projection-core"))
     .settings(Dependencies.core)
+    .settings(Test / parallelExecution := false)
```

Comment:
This is necessary until I can improve the in memory instruments so parallel test runs don't pollute each other's data.